docs/tutorials/03_ground_state_solvers.ipynb

```diff
@@ -15,7 +15,7 @@
     "\n",
     "<img src=\"aux_files/H2_gs.png\" width=\"200\">\n",
     "\n",
-    "In this tutorial we are going to discuss the ground state calculation interface of Qiskit Nature. The goal is to compute the ground state of a molecular Hamiltonian. This Hamiltonian can be electronic or vibrational. To know more about the preparation of the Hamiltonian, check out the Electronic structure and Vibrational structure tutorials. \n",
+    "In this tutorial we are going to discuss the ground state calculation interface of Qiskit Nature. The goal is to compute the ground state of a molecular Hamiltonian. This Hamiltonian can be electronic or vibrational. To know more about the preparation of the Hamiltonian, check out the [Electronic structure](01_electronic_structure.ipynb) and [Vibrational structure tutorials](02_vibrational_structure.ipynb). \n",
```

Comment:
As this is being updated should we take the chance to note, at least briefly in this tutorial, that we compute the electronic part, using the quantum algorithms like VQE, and with that then computing and returning the electronic part of the ground state we add in the constant nuclear repulsion part in the nature stack to form the total ground state energy result.